src/main/java/net/snowflake/client/core/OpaqueContextDTO.java

```diff
@@ -0,0 +1,23 @@
+package net.snowflake.client.core;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class OpaqueContextDTO {
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  private String base64Data;
```

Comment:
Why is it a string and not a series of bytes?

Reply:
In the future when we add some contents to `OpaqueContext` it will be base64 encoded to a `String`. To be consistent between GS and JDBC side, I use `String` type here and the JDBC side does not decode it.